etc/install_coq.sh

```diff
@@ -11,9 +11,9 @@ pushd "$DIR" 1>/dev/null || exit 1
 ROOT_DIR="$(git rev-parse --show-toplevel)"
 pushd "$ROOT_DIR" 1>/dev/null || exit 1
 
-if test ! -d .git
+if test ! ( -d .git -o -f .git )
```

Comment:
Don't use `-o` and `test !`, use the shell's `||` and `!` instead, so
```suggestion
if ! ( [ -d .git ] || [ -f .git ] )
```

Reply:
(or just use `-e`)